<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QDialog" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="MembersTab">
      <attribute name="title">
       <string>Members</string>
      </attribute>
      <attribute name="toolTip">
       <string>Member Management</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTableView" name="membersTable"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="addMemberButton">
             <property name="text">
              <string>Add Member</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeMemberButton">
             <property name="text">
              <string>Remove Member</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="InventoryTab">
      <attribute name="title">
       <string>Inventory</string>
      </attribute>
      <attribute name="toolTip">
       <string>Inventory Management</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableView" name="itemsTable">
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="addItemButton">
             <property name="text">
              <string>Add Item</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeItemButton">
             <property name="text">
              <string>Remove Item</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="UpgDowngTab">
      <attribute name="title">
       <string>Upgrade/Downgrade</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Suggested Membership Upgrades</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="UpgradeTable"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Suggested Membership Downgrades</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="DowngradeTable"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
